apps/nestjs-indexer/src/indexer/indexer.service.ts

```diff
@@ -44,13 +46,23 @@ export class IndexerService {
 
     this.logger.log('Starting indexer...');
 
+    const indexerStats = await prisma.indexerStats.findFirst({
+      orderBy: { lastBlockScraped: 'desc' },
+    });
+
+    const startingBlock = indexerStats
+      ? indexerStats.lastBlockScraped
+      : constants.STARTING_BLOCK;
+
+    console.log(startingBlock);
+
     const combinedFilter = this.combineFilters();
 
     this.client.configure({
       filter: combinedFilter,
       batchSize: 1,
       finality: v1alpha2.DataFinality.DATA_STATUS_ACCEPTED,
-      cursor: StarkNetCursor.createWithBlockNumber(constants.STARTING_BLOCK),
+      cursor: StarkNetCursor.createWithBlockNumber(Number(startingBlock)),
```

Comment:
Good, can we also add like a default value in case we have issue to get the last block scraped?